index.md

```diff
@@ -1,73 +1,52 @@
-# Please Don't Just Say Hello In Chat
+# Please Don't Just Say Hello in Chat
 
 ## TL;DR
 
-Modern chat communication should be done in a way that saves time for all
-participants and enables an efficient, asynchronous exchange of information.
-**Don't expect an answer if you just write "Hello" in chat**. Instead, phrase a
-message that allows your chat partners to provide a meaningful response as soon
-as they allocate time to read and answer your message.
+Modern chat communication should be done in a way that saves time for all participants and enables an efficient, asynchronous exchange of information. Don’t expect an answer if you just write “Hello” in chat. Instead, phrase a message that allows your chat partners to provide a meaningful response as soon as they allocate time to read and answer your message.
```

Comment:
Please keep the line wrap at 79 characters. This helps readability in plain text. Without wrap, horizontal *and* vertical scrolling as well as more eye movement is necessary.